src/transformers/models/table_transformer/modeling_table_transformer.py

```diff
@@ -550,12 +550,17 @@ def forward(
         return attn_output, attn_weights_reshaped
 
 
+TABLE_TRANSFORMER_ATTENTION_CLASSES = {
+    "eager": TableTransformerAttention,
+}
```

Comment:
Table transformer is almost the same as DETR, so it should be straightforward to add SDPA attention here as well with `Copied from` statement